scss/window/_theme.scss

```diff
@@ -18,14 +18,17 @@
     // Titlebar
     .k-window-titlebar {
         @include appearance( header );
+        color: $window-titlebar-text;
         border-color: inherit;
+        background: $window-titlebar-bg;
```

Comment:
The only piece which doesn't fit with that is we are not currently setting a border-color on the window - I will add the variable  though?